fe/fe-core/src/main/java/com/starrocks/metric/MaterializedViewMetricsRegistry.java

```diff
@@ -67,44 +63,40 @@ public void run() {
 
     // collect materialized-view-level metrics
     public static void collectMaterializedViewMetrics(MetricVisitor visitor, boolean minifyMetrics) {
-        GlobalStateMgr globalStateMgr = GlobalStateMgr.getCurrentState();
-        List<String> dbNames = globalStateMgr.getLocalMetastore().listDbNames();
-        for (String dbName : dbNames) {
-            Database db = GlobalStateMgr.getCurrentState().getDb(dbName);
-            if (null == db) {
+        MaterializedViewMetricsRegistry instance = MaterializedViewMetricsRegistry.getInstance();
+        for (Map.Entry<MvId, MaterializedViewMetricsEntity> e : instance.idToMVMetrics.entrySet()) {
+            IMaterializedViewMetricsEntity mvEntity = e.getValue();
+            if (mvEntity == null || mvEntity instanceof  MaterializedViewMetricsBlackHoleEntity) {
```

Comment:
what's that ? BlockHole ?

Reply:
If user disable mv metrics, all metrics will throw into `MaterializedViewMetricsBlackHoleEntity ` instead.